hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueFactory.kt

```diff
@@ -30,7 +30,15 @@ class HmppsQueueFactory(
       .map { (queueId, queueConfig) ->
         val sqsDlqClient = getOrDefaultSqsDlqClient(queueId, queueConfig, hmppsSqsProperties)
         val sqsClient = getOrDefaultSqsClient(queueId, queueConfig, hmppsSqsProperties, sqsDlqClient)
-          .also { subscribeToLocalStackTopic(hmppsSqsProperties, queueConfig, hmppsTopics) }
+          .also {
+            val queueArn = it.getQueueAttributes(
+              GetQueueAttributesRequest.builder()
+                .queueUrl("${hmppsSqsProperties.localstackUrl}/queue/${queueConfig.queueName}")
+                .attributeNames(QueueAttributeName.QUEUE_ARN).build()
+            ).get()
+              .attributes()[QueueAttributeName.QUEUE_ARN]
+            subscribeToLocalStackTopic(hmppsSqsProperties, queueConfig, queueArn!!, hmppsTopics)
+          }
```

Comment:
Hmmm it might be an idea to grab the topic arn on initialisation and save it in HmppsTopic. 🤔 

Not one for this PR though, this is fine 👍